java/com/google/flatbuffers/FlatBufferBuilder.java

```diff
@@ -845,6 +909,24 @@ private int dataStart() {
         return array;
     }
 
+    /**
+     * A utility function to return an InputStream to the ByteBuffer data
+     * from `start` to `start` + `length`.
+     *
+     * @param start Start reading at this offset.
+     * @param length How many bytes to read.
+     * @return An InputStream to the {@link #dataBuffer() data buffer} that begins at `start`
+     *         and can read up to `length` bytes.
+     * @throws IllegalArgumentException If the range of bytes is out of bounds.
+     */
+    public InputStream sizedInputStream(int start, int length) {
```

Comment:
Is there a reason why `sizedByteArray(int, int)` is public but maybe this should not be? I personally can't see any use to `sizedByteArray(int, int)` or `sizedInputStream(int, int)` that wouldn't be covered by `sizedByteArray()` and `sizedInputStream()` respectively. Perhaps I'm missing a use case that I haven't stumbled on yet. Was it intentional to make `sizedByteArray(int, int)` public?